code-challenges.test.js

```diff
@@ -49,12 +91,30 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
-// b) Create the function that makes the test pass.
+// Create function studySorter 
+// Use Object.values method to return array of the objects property values
+// Use .sort to sort the values and then use (a, b) => a - b to sort the values in ascending order
+
+const studySorter = (object) => {
+  return Object.values(object).sort((a, b) => a - b)
+  }
+
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("accumulatingArray", () => {
+  it("takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+      expect(accumulatingArray(accountTransactions1)).toEqual([100, 83, 60, 51])
```

Comment:
To level up your tests, consider moving your test variables into your tests just after the it statement and before the expect statement. Best practice calls on the variables being placed within the same scope in which they are used.